demos/rosinstall/src/quittable_listener/package.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<package>
+  <name>quittable_listener</name>
+  <version>0.0.0</version>
+  <description>The listener package</description>
+  <maintainer email="me@example.com">me</maintainer>
+  <license>TODO</license>
```

Comment:
That's fair, though note that this scaffolding is generated by `catkin_create_pkg`, not written by hand.